### ISSUE TITLE
Awesome Idea, can't get it to work as described

### ISSUE THREAD

**Issue by digitalprecision** [shinydevelopment/SimulatorStatusMagic#40]
Hello. I am having an issue with this project in that I can't get the status bar to work as described.

Here are the steps:

1. Clone repo
2. Open project (Using Xcode 8.0, if I try to use < 8.0 I get an error about 'null' on storyboard)
3. Ensure SimulatorStatusMagic is selected app
4. Select iPhone 6s Plus
5. Build
6. Simulator correctly loads the app
7. I don't see any 'button' other than the 'Apply clean status ....'
8. I click the button
9. Open my project
10. Build my project, it now shows in simulator
11. I don't see the signal indicator nor is the time set to 9:41am (see attached screenshot)
12. If I hit hardware -> home then re-open the SimulatorStatusMagic text in upper left hand corner disappears, but still no signal indicator

What am I missing?
![shiny_broken_ss](https://cloud.githubusercontent.com/assets/234718/18062511/0add5806-6ddc-11e6-8b70-fe00d0f41381.png)

**Comment by m-schmidt** [comment]
Which iOS version do you simulate in step 4? I got it to work with v9.3. For iOS10 supportthere is an open PR, but I didn't try it out.

**Comment by daveverwer** [comment]
Hey, I've seen the PR and I'll work on testing that now to see if it works.

**Comment by digitalprecision** [comment]
> Which iOS version do you simulate in step 4? I got it to work with v9.3. For iOS10 supportthere is an open PR, but I didn't try it out.

I'm a xcode / ios newb so you will have to ELI5. Is this the setting under SimulatorStatusMagic -> General -> Deployment Target? If so, it's currently set to 8.0.

**Comment by m-schmidt** [comment]
No, I meant the iOS version which "runs" in the simuator:

![xcode](https://cloud.githubusercontent.com/assets/1093186/18063711/6c60056a-6e2c-11e6-9e4e-e0a317aed4ad.png)

**Comment by digitalprecision** [comment]
Hmmm not seeing any versioning after any of the available emulators....
![screen shot 2016-08-29 at 12 10 27 pm](https://cloud.githubusercontent.com/assets/234718/18063860/acd4c4b4-6de1-11e6-99a8-056b92416dea.png)

**Comment by daveverwer** [comment]
Can you pull the new master and give it a new try please @digitalprecision? I'm assuming this was just caused by the fact that iOS 10 wasn't supported.

**Comment by digitalprecision** [comment]
> Can you pull the new master and give it a new try please @digitalprecision? I'm assuming this was just caused by the fact that iOS 10 wasn't supported.

d00d you got it! Now it's working like a champ. Thanks! The only thing I did notice is that it says SimulatorStatusMagic in the upper left hand corner (per my first screenshot), but if I click Hardware -> Home then re-open my app it disappears and everything works as expected.

Thanks for fixing!

**Comment by daveverwer** [comment]
Glad it's working :)

**Comment by m-schmidt** [comment]
You can click on the simulated device and select "Add additional simulators...". Configure them as wanted. You probably have to install simulator packages for other iOS versions in the Xcode preferences.

**Comment by digitalprecision** [comment]
> You can click on the simulated device and select "Add additional simulators...". Configure them as wanted. You probably have to install simulator packages for other iOS versions in the Xcode preferences.

Copy that. Thanks, I'll keep that in mind moving forward.
